features/FEATURE_LWIP/lwip-interface/lwipopts.h

```diff
@@ -83,13 +83,13 @@
 #ifdef LWIP_DEBUG
 #define TCPIP_THREAD_STACKSIZE      1200*2
 #else
-#define TCPIP_THREAD_STACKSIZE      1200
```

Comment:
@bulislaw I have a PR #4048 to address target specific options. It's ready for merge, but until that happens, this is the only way to do it right now.